lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -38,9 +40,19 @@ class AwsCompileScheduledEvents {
               }
               ScheduleExpression = event.schedule.rate;
               State = event.schedule.enabled ? 'ENABLED' : 'DISABLED';
+              Input = event.schedule.input || '';
+              InputPath = event.schedule.inputPath || '';
+
```

Comment:
Ok so here we will need to add something like this:

```javascript
if (Input && InputPath) {
  throw new this.serverless.classes.Error('You need to specify either "Input" or "InputPath" but not both');
}
```

Somebody with a better english please review the error message 😆  @eahefnawy @kbariotis